csi/node.go

```diff
@@ -113,6 +114,15 @@ func (s *OsdCsiServer) NodePublishVolume(
 	// Get volume encryption info from req.Secrets
 	driverOpts := s.addEncryptionInfoToLabels(make(map[string]string), req.GetSecrets())
 
+	// Parse storage class 'mountOptions' flags from CSI req
+	// flags from 'mountOptions' will be used as the only source of truth for Pure volumes upon mounting
+	if req.GetVolumeCapability() != nil && req.GetVolumeCapability().GetMount() != nil {
```

Comment:
Can you add a unit test for this in `node_test.go`? It can simply validate that the mount flags are being added to the expected mock mount request.

You can copy this test and edit it to this:
https://github.com/libopenstorage/openstorage/blob/master/csi/node_test.go#L410

```
		s.MockDriver().
			EXPECT().
			Mount(name, targetPath, map[string]string{"mount-opts": "here"}).
			Return(nil).
			Times(1),
```